### DISCUSSION TITLE
How to structure?

### DISCUSSION THREAD

**Discussion by joshualerickson** [gdal4al/gdal-examples#3] [Ideas]
Hey Community, thanks for your interest in this repo (star = interest 🤷‍♂️). I'm reaching out to see how we should structure this repo? I've got a few ideas but I'd like to hear from the group. I like having the lists of links within a category but think a more structured setup might benefit users and developers with examples. Couple things:

* Start a bookdown with examples per category. Within categories have examples within different use cases, e.g. DEM, imagery, vsi/vrt, etc.   
* Link/tag these examples to the list on the splash page.  So when you click on a 'topic' (R -> DEM) it links to the bookdown page examples.  
* Use the issues/discussions/PR as development of the bookdown/lists.  

There's already a lot out there so reinvention is likely to happen but I think that having 'tricks of the trade' collated will be nice for new users as well as the experts.  

Just watched @mikejohnson51 talk on vsi/vrt and was in awe of the climateR package he has developed! This package could provide so many examples within R and Python. 

None of this is fully formed just stream of consciousness but hoping to work together and find a spot for the bright and thoughtful GDAL community. 

@mdsumner @brownbag @ambarja @wiesehahn @jackholbrook

**Comment by sjrh2** [comment]
Hi there! I found this repo by chance. Seems like a great idea and I’m happy to help. I’m not familiar with what the most popular use cases are but I would be happy to contribute CLI/Python and python-gdal used through the subprocess module. A bookdown page sounds clean and neat.

**Reply by joshualerickson** [reply]
That is great to hear! I'll try and get something going in the next few days to help with development and such. I think anything goes in terms of use cases :). I'll create a folder called 'examples' and then within there we can categorize based on language use case. 

Might be a little chaotic at first but really appreciate your help!
